client/dashboard/profile-wizard/style.scss

```diff
@@ -6,6 +6,25 @@
 	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell,
 		'Helvetica Neue', sans-serif;
 
+	.woocommerce-profile-wizard__container a {
+		color: #beabce;
```

Comment:
You can use `$muriel-purple-200` for this.

Reply:
Nice- switched this over to `$muriel-gray-600` to match new designs.